firebase-common/gradle.properties

```diff
@@ -1,3 +1,3 @@
-version=20.4.1
-latestReleasedVersion=20.4.0
+version=20.4.2
+latestReleasedVersion=20.4.1
```

Comment:
is this expected?

Reply:
yes

Reply:
there is already an admrt push to release firebase-common at 20.4.1 .. This is essentially updating that